amy/fiscal/views.py

```diff
@@ -248,6 +248,9 @@ class MembershipMembers(
 ):
     permission_required = "workshops.change_membership"
 
+    def get_formset(self, *args, **kwargs):
+        return modelformset_factory(Member, MemberForm, *args, **kwargs)
```

Comment:
nit: This appears to just have been moved no changes. probably isn't necessary

Reply:
Good catch! This was moved to reflect the order of methods in a "sibling" view - `MembershipTasks`.